ra_aid/project_info.py

```
@@ -0,0 +1,123 @@
+"""Module providing unified interface for project information."""
+
+from dataclasses import dataclass
+from typing import List, Optional
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)